src/main/java/com/spotify/github/v3/clients/PullRequestClient.java

```diff
@@ -173,6 +175,46 @@ static PullRequestClient create(
     return github.post(path, jsonPayload, Review.class);
   }
 
+  /**
+   * List pull request requested reviews.
+   *
+   * @param number pull request number
+   * @return list of reviews
+   */
+  public CompletableFuture<ReviewRequests> listReviewRequests(final int number) {
+    final String path = String.format(PR_REVIEW_REQUESTS_TEMPLATE, owner, repo, number);
+    log.debug("Fetching pull request requested reviews from " + path);
+    return github.request(path, LIST_REVIEW_REQUEST_TYPE_REFERENCE);
+  }
+
+  /**
+   * Requests a review for a pull request.
+   *
+   * @param number pull request number
+   * @param properties properties for reviewing the PR, such as reviewers and team_reviewers.
+   * @see "https://docs.github.com/en/rest/reference/pulls#request-reviewers-for-a-pull-request"
+   */
+  public CompletableFuture<PullRequest> requestReview(final int number, final RequestReviewParameters properties) {
+    final String path = String.format(PR_REVIEW_REQUESTS_TEMPLATE, owner, repo, number);
+    final String jsonPayload = github.json().toJsonUnchecked(properties);
+    log.debug("Requesting reviews for PR: " + path);
+    return github.post(path, jsonPayload, PullRequest.class);
+  }
+
+  /**
+   * Remove a request for review for a pull request.
+   *
+   * @param number pull request number
+   * @param properties properties for reviewing the PR, such as reviewers and team_reviewers.
+   * @see "https://docs.github.com/en/rest/reference/pulls#request-reviewers-for-a-pull-request"
+   */
+  public CompletableFuture<Void> removeRequestedReview(final int number, final RequestReviewParameters properties) {
```

Comment:
it'd be nice to add a test for this case, so we can get the new top-level `delete(path, jsonPayload)` method tested as well